osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -105,7 +108,7 @@ private void load(FrameworkConfigManager config, OsuConfigManager osuConfig, Gam
                     LabelText = GraphicsSettingsStrings.Resolution,
                     ShowsDefaultIndicator = false,
                     ItemSource = resolutions,
-                    Current = sizeFullscreen
+                    Current = currentResolution
```

Comment:
This is forcing two unrelated settings (fullscreen size and windowed size) onto the same dropdown. I would make two dropdowns and hide and show them depending on the current screen mode.

Reply:
> This is forcing two unrelated settings (fullscreen size and windowed size) onto the same dropdown. I would make two dropdowns and hide and show them depending on the current screen mode.

Ok, I separated the dropdowns and source lists (435cd272eaa3f291921e804255f33a3c53d92da4).